.travis.yml

```diff
@@ -1,38 +1,37 @@
 language: ruby
-sudo: false
 cache: bundler
 
 rvm:
-  - 2.5.3
-  - 2.4.5
-  - 2.3.8
-  - 2.2.9
-  - 2.1.10
+  - 2.7
+  - 2.6
+  - 2.5
+  - 2.4
+  - 2.3
+  - 2.2
+  - 2.1
```

Comment:
This should pick the latest patch version, so it doesn't have to be micro-managed with every patch release.